juniper_official/Node-Exporter/check-disk-read-usage.rule

```diff
@@ -0,0 +1,260 @@
+            field time-seconds-used-percentage {
+                formula {
+                    user-defined-function {
+                        function-name used-percentage;
+                        argument total "$frequency";
+                        argument used "$diff-time-seconds";
+                    }
+                }
+                type float;
+                description "Percentage of disk read time";
+            }
+            /*
+             * Anomaly detection logic.
+             */			
+            trigger bytes-total-increase {
+                frequency 1offset;
+                term exceeds-threshold {
+                    when {
+                        greater-than-or-equal-to "$diff-bytes" "$threshold-increase" {
+                            time-range 3offset;
+                            any;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "bytes-total-increase $diff-bytes for $disk-name exceeds threshold";
+                        }
+                    }
+                }
+                term normal {
+                    then {
+                        status {
+                            color green;
+                            message "bytes-total-increase $diff-bytes for $disk-name  is normal";
+                        }
+                    }
+                }
+            }
+            trigger completed-used-percentage {
+                frequency 1offset;
+                term exceeds-threshold {
+                    when {
+                        greater-than-or-equal-to "$completed-used-percentage" "$threshold" {
+                            time-range 3offset;
+                            any;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
+                            message "completed-used-percentage \"$completed-used-percentage\" for $disk-name exceeds threshold";
+                        }
+                    }
+                }
+                term normal {
+                    then {
+                        status {
+                            color green;
+                            message "completed-used-percentage \"$completed-used-percentage\" for $disk-name is normal";
+                        }
+                    }
+                }
+            }
+            trigger time-seconds-used-percentage {
+                frequency 1offset;
+                term exceeds-threshold {
+                    when {
+                        greater-than-or-equal-to "$time-seconds-used-percentage" "$threshold" {
+                            time-range 3offset;
+                            any;
+                        }
+                    }
+                    then {
+                        status {
+                            color red;
```

Comment:
Should we be flagging disk reads as red at all?
I mean, at best we can indicate that it is anomalous behavior.. But it isn't really errant behavior though.
Should this be yellow?

Reply:
Changed color to yellow.